content/docs/v3.4/op-guide/clustering.md

```diff
@@ -268,7 +268,7 @@ This will cause each member to register itself with the custom etcd discovery se
 
 #### Public etcd discovery service
 
-If no existing cluster is available, use the public discovery service hosted at `discovery.etcd.io`.  To create a private discovery URL using the "new" endpoint, use the command:
+If no exiting cluster is available, use the public discovery service hosted at `discovery.etcd.io`.  To create a private discovery URL using the "new" endpoint, use the command:
```

Comment:
why this change?

Reply:
Sorry, good catch, I'll fix this. This is an artifact of merging in from the `next` directory. I'll fix it there too.